includes/class-ssp-frontend.php

```diff
@@ -1716,251 +1782,45 @@ public function podcast_episode ( $episode_id = 0, $content_items = array( 'titl
 			 * @todo Add filters
 			 * @todo Add settings pages to customize layout / colours
 			 */
-			$meta = $this->episode_meta_details( $episode_id, $episode_context, true );
-			$file = $this->get_enclosure( $episode_id );
 
 			if( 'mini' !== $style ){
 				if( 'large' == $style ){
-					ob_start();
```

Comment:
Removed all this legacy HTML5 player code in favor of using the `media_player()` method, far more predictable.